Assignment/solution_student/css/style.css

```diff
@@ -31,19 +31,120 @@ html {
 /* BASE STYLES
 /*******************************************/
 body {
-  color: black;
+  color: #121212;
   font-size: 16px;
-  background: white;
+  background-color: #f0efef;
+  font-weight: 300px;
+  font-family: 'open Sans', sans-serif;
+
 }
 
 /******************************************
 /* LAYOUT
 /*******************************************/
 
+
 /******************************************
 /* PAGES
 /*******************************************/
 
 /******************************************
 /* SECTIONS
-/*******************************************/
\ No newline at end of file
+/*******************************************/
+
+/* Site Header */
+header {
+  background-color: #033048;
+  height: 100px;
+  padding: 40px 0;
+  
+
+}
+header h1{
+	display: inline-block;
+	font-size: 24px;
+	margin: 0;
+	color: #f0efef;
+	margin: 0;
+	padding-left: 15%;
+	float: left;
+}
+nav ul{
+	padding-right: 10%;
+	padding-top: 1em;
+	margin: 0px;
+	float: right;
+}
+nav li{
+	display: inline-block;
+
+	font-weight: 500;
+	color: #f9e42e;
+	margin: 0px
+}
+a{
+	color: yellow;
+	text-decoration: none;
+}
+
+#adUnit{
+	background: url('images/ad-unit.png');
+    /*background-size: cover;*/
+}
+
+
+
+
+/* Main Body */
+.firstletter {
+	float: left;
+	font-size: 60px;
+	font-weight: 500;
+	line-height: 60px;
+	padding: 0px 5px; 
+}
+.readMore{
+	text-align: right;
+	padding-bottom: 15px
```

Comment:
Oops! You missed the `;` at the end of the line, which is screwing up the next line!